www/docs/scm/gitlab.md

```diff
@@ -18,8 +18,8 @@ env_files:
 
 ## GitLab Enterprise or private hosted
 
-You can use GoReleaser with GitLab Enterprise by providing its URLs in
-the `.goreleaser.yml` configuration file:
+You can use GoReleaser with GitLab Enterprise by providing its URLs in the
+`.goreleaser.yml` configuration file. This takes a normal string or a template value.
 
 ```yaml
```

Comment:
**thought:** Not sure if we want to give a commented example here that it can support templates as well like below

```
# .goreleaser.yml
gitlab_urls:
  # api: "{{ .ENV. CI_API_V4_URL }}
  api: https://gitlab.mycompany.com/api/v4/
  download: https://gitlab.company.com
  # set to true if you use a self-signed certificate
  skip_tls_verify: false
```

Reply:
no need IMHO, docs are clear enough 🙏